src/store.js

```diff
@@ -0,0 +1,67 @@
+import { createStore } from 'redux';
+
+const SET_TODOS = 'set todos';
+const CHANGE_LOADING = 'change loading';
```

Comment:
```suggestion
const CHANGE_LOADING = 'CHANGE_LOADING';
```
прийнято писати назву action також в uppercase